From 26434ef46761b6e1303f394c32031057b7fec4fd Mon Sep 17 00:00:00 2001
From: Mark McLoughlin <markmc@redhat.com>
Date: Tue, 25 Oct 2011 06:19:27 +0100
Subject: [PATCH 13/44] Add missing author

Add Aaron Lee to Authors for the previous patch which somehow got
past Jenkins.

Change-Id: I671685f261f687bb4f9e3c39db25c57115927b69
---
 Authors |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Authors b/Authors
index 8d6837e..f3962c8 100644
--- a/Authors
+++ b/Authors
@@ -1,3 +1,4 @@
+Aaron Lee <aaron.lee@rackspace.com>
 Adam Gandelman <adamg@canonical.com>
 Adam Johnson <adjohn@gmail.com>
 Alex Meade <alex.meade@rackspace.com>
-- 
1.7.6.5

